lib/player.rb

```diff
@@ -0,0 +1,18 @@
+class Player
+  attr_reader :name,
+              :board
+
+  def initialize(name, board)
+    @board = board
+    @name = name
+  end
+
+  def combined_ship_health(health)
+    cruiser + submarine
+  end
+
+  def has_lost?
+    #require 'pry'; binding.pry
```

Comment:
My prys fell out of my pocket and I left them laying around.  I'm going to need to clean that up.  Cheers!